ccx_messaging/engines/multiplexor_engine.py

```diff
@@ -49,18 +51,22 @@ def process(self, broker: Broker, path: str) -> set[str]:
         self.logger.info("Opening %s for multiplexing", path)
         self.fire("pre_extract", broker, path)
 
-        with tarfile.open(path) as tf:
-            self.fire("on_extract", None, broker, tf)
+        try:
+            with tarfile.open(path) as tf:
+                self.fire("on_extract", None, broker, tf)
+
+                filenames = tf.getnames()
 
-            filenames = tf.getnames()
+                marks = set()
+                for file_to_find, mark in self.filters.items():
+                    if file_to_find in filenames:
+                        marks.add(mark)
 
-            marks = set()
-            for file_to_find, mark in self.filters.items():
-                if file_to_find in filenames:
-                    marks.add(mark)
+            if not marks:
+                marks.add("DEFAULT")
 
-        if not marks:
```

Comment:
is this delete safe enough?

Reply:
It is just a few lines above now, GitHub diff didn't detect it well, but those lines are in line 65 on the new version